docs/articles/expensify-classic/workspaces/reports/Scheduled-Submit.md

```diff
@@ -29,10 +29,15 @@ For individuals or employees: To enable Scheduled Submit on your individual work
 
 **Manually**: An open report will be created, and expenses will be added to it automatically. However, it's important to note that the report will not be submitted automatically; manual submission of reports will be required.This is a great option for automatically gathering all an employee’s expenses on a report for the employee’s convenience, but they will still need to review and submit the report.
 
+**Instant Submit**: Expenses are automatically added to a report in the Processing state, and all expenses will continue to accumulate on one report until it is Approved or Reimbursed. This removes the need to submit expenses, and Processing reports can either be Reimbursed right away (if Submit and Close is enabled), or Approved and then Reimbursed (if Submit and Approve is enabled) by a workspace admin.
+
 # Deep Dive
 
 ## Schedule Submit Override
 If Scheduled Submit is disabled at the group workspace level or configured the frequency as "Manually," the individual workspace settings of a user will take precedence and be applied. This means an employee can still set up Scheduled Submit for themselves even if the admin has not enabled it. We highly recommend Scheduled Submit as it helps put your expenses on auto-pilot!
 
 ## Personal Card Transactions
 Personal card transactions are handled differently compared to other expenses. If a user has added a card through Settings > Account > Credit Card Import, they need to make sure it is set as non-reimbursable and transactions must be automatically merged with a SmartScanned receipt. If transactions are set to come in as reimbursable or they aren’t merged with a SmartScanned receipt, Scheduled Submit settings will not apply.
+
+## A note on Instant Submit 
```

Comment:
Same as above. I think it should be `Instantly` instead of `Instant Submit`

Reply:
Discussed on [slack](https://expensify.slack.com/archives/C02QSAC6BJ8/p1714589267255109) since Steph is OOO and we're going with Instantly instead of Instant Submit.